components/NavBar.vue

```diff
@@ -1,249 +1,66 @@
-      "
-      @update:show="isHamburgerShown = $event"
-    >
-      <template #trigger>
-        <div
-          v-show="isSmallDisplay"
-          class="flex mr-5 items-center relative"
-        >
-          <div
-            v-if="isHamburgerShown"
-            class="fixed z-50 inset-0 w-full h-full"
-            role="dialog"
-            aria-modal="true"
-          >
-            <div
-              class="backdrop-blur-sm bg-gray-400/20 w-full h-full"
-              aria-hidden="true"
-            ></div>
-          </div>
-          <button
-            class="text-right text-gray-700 text-xl relative z-60"
-            @click="isHamburgerShown = !isHamburgerShown"
-          >
-            <Icon :name="isHamburgerShown ? 'ri:close-fill' : 'ri:menu-fill'" />
-          </button>
-        </div>
-      </template>
-      <div class="px-6 py-5 text-base max-w-xs w-full">
-        <button
-          class="text-right text-gray-700 text-xl absolute top-3 right-5"
-          @click=";[hamburgerMenu?.setShow(false), (isHamburgerShown = false)]"
-        >
-          <Icon
-            v-if="isHamburgerShown"
-            name="ri:close-fill"
-          />
-        </button>
-        <slot name="collapsed" />
-      </div>
-    </n-popover>
-
-    <!-- Main menu -->
-    <div v-show="!isSmallDisplay">
-      <slot>
-        <div class="space-x-14">
-          <span class="text-primary-600 text-lg font-semibold">Library</span>
-          <span class="text-gray-400 text-lg font-semibold">Browse</span>
-          <div class="inline">
-            <span class="text-gray-400 text-lg font-semibold">
-              Subscriptions
-            </span>
-            <div class="inline-block align-top pl-0.5 -mt-1">
-              <Icon
-                name="ri:checkbox-blank-circle-fill"
-                class="text-primary-600 text-xs"
-              />
+    <n-page-header class="pl-20 pt-5">
+        <template #avatar>
+            <JabrefLogo class="w-10 flex-none"/>
+        </template>
+        <template #title>
+            <a href="/" class="ml-3 flex-1 text-gray-900 text-2xl font-semibold lg:inline-block hidden">JabRef</a>
+        </template>
+        <template #extra>
+            <div class="flex justify-evenly pr-20">
+                <nav class='justify-evenly'>
+                    <t-nuxtlink
+                    active-class="text-gray-400 hover:text-primary-600 text-lg font-semibold"
+                    class="text-gray-400 hover:text-primary-600 text-lg font-semibold"
+                    to="https://docs.jabref.org/"
+                    >Docs</t-nuxtlink
+                    >
+                    <n-divider vertical />
+                    <t-nuxtlink
+                    active-class="text-gray-400 hover:text-primary-600 text-lg font-semibold"
+                    class="text-gray-400 hover:text-primary-600 text-lg font-semibold"
+                    to="https://builds.jabref.org/"
+                    >Builds</t-nuxtlink
```

Comment:
I don't think the nightly builds are that relevant.

Reply:
Ah, I'll remove them then, much thanks+